ci/gpuci/run.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# Copyright (c) 2020, NVIDIA CORPORATION.
+####################################
+# ccache fork run script #
+####################################
+set -ex
+
+export HOME=${WORKSPACE}
+export PATH="/opt/conda/bin:$PATH"
+source activate base
+conda config --set ssl_verify false
+conda install -k conda-build conda-verify anaconda-client
```

Comment:
I'm not sure if this is necessary, but in the past I had issues with the `noarch` builds when I didn't use the latest `conda-build` from `conda-forge`. Should the `conda-build` here also be installed from `conda-forge`?

Reply:
Not entirely sure but I'm leaning towards keeping it as is to remain as similar to gpuci-tools as possible since they are both part of the gpuci channel.